docs/pt/Accessing-Alluxio-from-Zeppelin.md

```diff
@@ -10,4 +10,4 @@ priority: 4
 de dados interativa. A versão anterior de Alluxio (Tachyon 0.8.2) foi 
 [integrada](https://github.com/apache/incubator-zeppelin/blob/master/docs/interpreter/tachyon.md) 
 com o Zeppelin e, em breve, nós iremos atualizar a integração para trabalhar com a versão mais 
-atual do Alluxio.
\ No newline at end of file
+atual do Alluxio.
```

Comment:
Github seems to think there is since it's showing it here. You could run `git checkout origin/master docs/pt/Accessing-Alluxio-from-Zeppelin.md` to revert the file to the version in the master branch, then you could commit the revision.